app/js/LearnedWordsManager.js

```diff
@@ -24,6 +24,22 @@ const LearnedWordsManager = {
     )
   },
 
+  unlearnWord(userToken, word, callback) {
+    if (callback == null) {
+      callback = () => {}
+    }
+    mongoCache.del(userToken)
```

Comment:
Not related to this change, but this cache is breaking HA solutions.

Are you running a single instance of `spelling` in production?

Reply:
You're right about that. We may have been, but it's possible that's changed and explains another issue that we're seeing. Thanks for the spot.

Reply:
We're aware of that, cache will likely be gone soon.